iothub/service/src/AmqpServiceClient.cs

```diff
@@ -37,21 +37,25 @@ internal sealed class AmqpServiceClient : ServiceClient
 
         private int _sendingDeliveryTag;
 
-        public AmqpServiceClient(IotHubConnectionString iotHubConnectionString, bool useWebSocketOnly, ServiceClientTransportSettings transportSettings, ServiceClientOptions options)
+        public AmqpServiceClient(
+            IotHubConnectionProperties connecitonProperties,
+            bool useWebSocketOnly,
+            ServiceClientTransportSettings transportSettings,
+            ServiceClientOptions options)
         {
-            var iotHubConnection = new IotHubConnection(iotHubConnectionString, AccessRights.ServiceConnect, useWebSocketOnly, transportSettings);
+            var iotHubConnection = new IotHubConnection(connecitonProperties, AccessRights.ServiceConnect, useWebSocketOnly, transportSettings);
             Connection = iotHubConnection;
             OpenTimeout = IotHubConnection.DefaultOpenTimeout;
             OperationTimeout = IotHubConnection.DefaultOperationTimeout;
             _sendingPath = "/messages/deviceBound";
```

Comment:
Why not initialize this inline where the field is declared? It could be a const then.

Reply:
Sure. Does anyone have context on if this can be a different value at all for any constructor?